django-stubs/db/models/fields/__init__.pyi

```diff
@@ -206,9 +206,9 @@ class Field(RegisterLookupMixin, Generic[_ST, _GT]):
     def formfield(
         self,
         form_class: type[forms.Field] | None = ...,
```

Comment:
```suggestion
        form_class: type[Field] | None = ...,
```

Reply:
@intgr No, these are different fields!

`forms.Field` is a form field. The local `Field` is a model field.

This change now breaks the annotation correctness. I'll open a supplemental PR to fix it now.

Reply:
Oops, sorry for messing that up. And thanks for responding so quickly.

Sadly this screw-up is now part of the django-stubs 4.2.5 release. I'll probably wait a few days to see if any other bugs turn up, and make another hotfix release.